documentation/api/guides/deploy.md

```diff
@@ -0,0 +1,42 @@
+# Deployment runbook
+
+```{tip}
+For more information on how deployments work, please see the [general deployment guide](/general/deployment.md).
+```
+
+1. Record and verify the commit `sha` of the production site at
+   [https://api.openverse.engineering/v1](https://api.openverse.engineering/v1).
+1. Visit
+   [https://api-staging.openverse.engineering/version](https://api-staging.openverse.engineering/version)
+   and
+   [the api package directory](https://github.com/wordpress/openverse/pkgs/container/openverse-api).
+   Verify that the commit `sha` live on the staging site is also tagged with
+   `latest` in the package directory.
+
+   ```{figure} /_static/package_directory_example.png
+   ---
+   class: with-border
+   ---
+   ```
```

Comment:
That's a very verbose way to add borders 😆. Not ideal.

Reply:
Copied this from one of my previous PRs, forgetting the `![]()` markdown syntax works here. Fixed.